Src/WitsmlExplorer.Api/appsettings.Development.json

```diff
@@ -1,4 +1,8 @@
 {
+"MongoDb": {
```

Comment:
this should not be part of the commit

Reply:
this file has been reverted to original version